### DISCUSSION TITLE
IR Receiver

### DISCUSSION THREAD

**Discussion by JohnFFlanagan** [DCC-EX/CommandStation-EX#432] [Ideas]
Are there any plans for an IR Receiver driver?

**Comment by Asbelos** [comment]
Realistically no.. ir receivers require a large amount of high frequency sampling and there are serious disadvantages relating to line of sight and directionality. 
In essence, wifi is so ubiquitous, more flexible and more reliable, there is little point in introducing an ir receiver.

**Comment by JohnFFlanagan** [comment]
Makes sense to me.  I was using IR for something else.  Dedicated RP2040
right now so sampling is not an issue.

Thanks for the reply. Let's close this thread if you agree.
